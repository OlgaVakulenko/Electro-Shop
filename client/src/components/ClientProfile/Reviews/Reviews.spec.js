import React from 'react';
import { shallow } from 'enzyme';

import { Reviews } from './Reviews';

describe('The test of Reviews component', () => {
  it('Reviews was rendered', () => {
    const rendered = shallow(
      <Reviews />,
    );

    expect(rendered).toMatchSnapshot();
  });
});
