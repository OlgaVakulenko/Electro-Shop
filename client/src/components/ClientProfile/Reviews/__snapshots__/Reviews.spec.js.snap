// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The test of Reviews component Reviews was rendered 1`] = `
<Fragment>
  <WithStyles(ForwardRef(ExpansionPanel))
    expanded={false}
    onChange={[Function]}
  >
    <WithStyles(ForwardRef(ExpansionPanelSummary))
      aria-controls="panel5bh-content"
      expandIcon={<Memo(ExpandMoreIcon) />}
      id="panel5bh-header"
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-heading-2"
        style={
          Object {
            "color": "#6A86E8",
          }
        }
      >
        My Reviews
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(ExpansionPanelSummary))>
    <WithStyles(ForwardRef(ExpansionPanelDetails))
      className="makeStyles-expansionPanelDetails-1"
    >
      <WithStyles(ForwardRef(Typography))>
        Nunc vitae orci ultricies, auctor nunc in, volutpat nisl. Integer sit amet egestas eros, vitae egestas augue. Duis vel est augue.
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(ExpansionPanelDetails))>
  </WithStyles(ForwardRef(ExpansionPanel))>
</Fragment>
`;
